app/models/scale.rb

```diff
@@ -1,53 +0,0 @@
-class Scale < ApplicationRecord
-  has_many :scalings, :dependent => :destroy
-
-  # attr_accessible :image_name, :key, :pos, :title
-  alias_attribute :name, :title
-
-  default_scope -> { order("pos ASC") }
-
-  validates_presence_of :title,:key,:image_name
-  validates_uniqueness_of :title,:key,:image_name
-
-  after_destroy :remove_annotations
```

Comment:
could there be annotations left hanging around after the scales related tables are dropped, and does it matter ?

Reply:
Yeah you're right, I'll add a task to clear them out